### ISSUE TITLE
Show hidden folder if yazi was opened in a subdirectory of that folder

### ISSUE THREAD

**Issue by Rolv-Apneseth** [sxyazi/yazi#411]
### Is your feature request related to a problem? Please describe

I opened `yazi` within a `.config` subdirectory (using `stow` so e.g. `progam/.config/program`) and of course since hidden files weren't being shown, trying to navigate backwards had an empty column where the `.config` directory would be.

This is very minor and I will personally just keep hidden files shown all the time, but for a minute there I was confused when first using the program.

### Will you be willing to contribute this feature?

- [ ] Yes, I'll give it a shot

### Describe the solution you'd like

Would it be possible to show a hidden directory, if `yazi` was opened in a subdirectory? This would allow navigating away and back to the original `cwd`, even if it is a subdirectory of a hidden folder, and avoid having an empty column.

**Comment by DrVoidest** [comment]
Can't you just hit "." to show hidden files or does it not work in subdirectories?

**Comment by Rolv-Apneseth** [comment]
Yeah probably (I just have it show hidden files by default now), I only opened this cause I was confused the first time I used Yazi but it's by no means necessary

**Comment by majutsushi** [comment]
I do think it would be nice to see the current directory in the left pane even if it is a hidden one (and hidden files are turned off). I've also occasionally been a bit confused briefly by this, especially when I've wanted to go to the parent directory for a moment to check something and then back again.
